### PR TITLE
fix: handle tokens without metadata

### LINKED ISSUE #208: Bridge fails for tokens without metadata (symbol and name)
image

Happens on staging right now. Registered an ERC721 which is not implementing ERC721Metadata (see [code](https://rinkeby.etherscan.io/address/0x611bbdb910e031a69d513ff6cc995e053fbea7b7#code)). Bridge fails trying to [read metadata](https://github.com/leapdao/bridge-ui/blob/master/src/stores/utils.ts#L68)

### PR DESCRIPTION
**Pull request by troggy**
For tokens without metadata implemented, show first bytes of the address as a symbol and color as a name.
Resolves #208

### CODE REVIEW

**Comment by troggy** [comment]
@sunify pushed a proper fix 😊

File: src/stores/utils.ts [hunk 1]
```diff
@@ -65,9 +68,9 @@ export const tokenInfo = (
   color: number
 ): Promise<[string, string, string]> => {
   return Promise.all([
-    tokenValue(web3, token, 'symbol'),
+    tokenValue(web3, token, 'symbol') || token.options.address.substring(0, 5),
```
Comment (eshavkun) [inline src/stores/utils.ts]: Maybe it's better to get a substring starting from 2 (as the first 2 symbols are always `0x`)?

Comment (troggy) [reply src/stores/utils.ts]: often times symbol is written near the balance which is numeric value. Having two decimals written one after another will look confusing. I'm not even sure 0x is not confusing as well :)

Comment (eshavkun) [reply src/stores/utils.ts]: `0x` would still be kinda confusing I think. Than maybe use color (the same as with name), smth like `TKN${color}`? Or if you want to stick with address, use only letters smth like `token.options.address.replace(/[^a-fA-F]/gi,'').substring(0,4)`?

Comment (sunify) [reply src/stores/utils.ts]: How about `COLOR_N`?

**Comment by troggy** [comment]
@sunify @eshavkun proposed arguably better schema for default names

**Pull request merged**